src/site/_includes/js/eligibility.js

```diff
@@ -2777,11 +2779,21 @@ function hudVashResult(input) {
     cnst.hudvash.ANNUAL_INCOME_LIMIT_ADDL_PERSON);
   const incomeLimit = grossLimit.getLimit(input.householdSize);
   const underIncomeLimit = le(grossIncome(input), incomeLimit);
+  const meetsImmigrationReq = or(
+    input.citizen,
+    validImmigration(input));
 
+  // https://www.federalregister.gov/documents/2024/08/13/2024-17957/section-8-housing-choice-vouchers-revised-implementation-of-the-hud-veterans-affairs-supportive
+  program.addCondition(
+    new EligCondition('Be a U.S. citizen or qualified immigrant',
```

Comment:
You can document this condition using this link: https://www.ecfr.gov/current/title-24/subtitle-B/chapter-IX/part-982/subpart-E/section-982.201#p-982.201(a)